src/plugins/spark/components/ActuatorState.vue

```diff
@@ -37,14 +37,18 @@ export default class ActuatorState extends Vue {
 
 <template>
   <div>
-    <q-toggle v-if="stringState !== 'Unknown'" v-model="boolState"/>
+    <q-toggle v-if="stringState !== 'Unknown'"
+              v-model="boolState" 
+              checked-icon="mdi-power-on"
```

Comment:
These icons don't add anything. A toggle itself is already widely understood as on/off.

Reply:
An alternative approach might be to use https://quasar-framework.org/components/button-toggle.html. This could be a more natural implementation for the three possible states (on/off/unknown)